cmd/listClients.go

```diff
@@ -0,0 +1,30 @@
+/*
+Copyright © 2022 Nethermind hello.nethermind.io
+
+*/
+package cmd
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/NethermindEth/1Click/configs"
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+)
+
+// listClientsCmd represents the listClients command
+var listClientsCmd = &cobra.Command{
+	Use:   "listClients",
+	Short: "List supported clients",
+	Long:  `List supported clients for execution and consensus engines`,
+	Run: func(cmd *cobra.Command, args []string) {
+		log.Info("Listing supported clients")
+		fmt.Printf("Eth1 Clients: \n%s\n\n", strings.Join(configs.Eth1ClientsSupported, ", "))
```

Comment:
Why fmt.Print ?

Reply:
I decided to use logs after writing this. Yeah, will not use fmt.Print here